ApollosProject/apollos-embeds
# (Web) Create Series layout

## Basecamp Scope
[(Web) Create Series layout](https://3.basecamp.com/3926363/buckets/27088350/todos/6419891226)

## What was done?

1. Add `ContentSeriesSingle` to render `ContentSeriesContentItem`
2. Added a component to truncate description text if they are too long
NEW:
3. Removed margins from `MediaItem`
4. Added Share button back

~~Note: Doesn't have share button which was on `ContentSingle` since it was excluded in Figma~~

## How to test?

1. Click on a `ContentSeriesContentItem`
2. Should follow design laid out in Figma: https://www.figma.com/file/aiXp20H5yMdkOWyXOvNNqh/Web-Embeds?type=design&node-id=149-36531&mode=dev
5. Text should truncate in description if too long (can see in video below)

https://github.com/ApollosProject/apollos-embeds/assets/68402088/c7f41f1a-c77a-45b7-8549-b6ab54b27376

UPDATED VIDEO:


https://github.com/ApollosProject/apollos-embeds/assets/68402088/516fdac4-566c-4665-a61c-2bab654ebbd7
